core/templates/components/forms/custom-forms-directives/image-receiver.component.html

```diff
@@ -1,46 +1,29 @@
+  <div class="image-receiver-text">{{ 'I18N_DIRECTIVES_DRAG_IMAGE_HERE' | translate }}</div>
   <span *ngIf="errorMessage"
-        class="image-uploader-error-message e2e-test-upload-error">
+        class="image-receiver-error-message e2e-test-upload-error">
     <i class="fas fa-exclamation-triangle"></i>
     {{ errorMessage }}
   </span>
-  <span class="image-uploader-license-warning" *ngIf="!isBlogPostThumbnailUploader" tabindex="0">
+  <span class="image-receiver-license-warning" tabindex="0">
     Before uploading, please ensure that your image:<br>
     <ul class="image-upload-instructions">
-      <li>Is in .svg, .png or .jpg format</li>
-      <li>Has a size smaller than 100 KB</li>
+      <li>{{ getAllowedImageFormatsString() }}</li>
+      <li>Has a size smaller than {{ maxImageSizeInKB }} KB</li>
       <li>Has no watermarks and is compatible with the <a tabindex="0" class="license-term-link" href="/license" [smartRouterLink]="'/' + licenseUrl" target="_blank" rel="noopener">license terms</a> of the site</li>
     </ul>
   </span>
-  <label for="image-file-input" class="image-uploader-upload-label-button" *ngIf="!isBlogPostThumbnailUploader">{{ 'I18N_DIRECTIVES_UPLOAD_A_FILE' | translate }}</label>
-  <!-- The onclick="this.imageInputRef.nativeElement.value=null" attribute resets the value of the input so that uploading the same image twice in a row works. This fix is only needed for chrome. Reference: stackoverflow.com/q/12030686 -->
+  <label for="image-file-input" class="image-receiver-upload-label-button">{{ 'I18N_DIRECTIVES_UPLOAD_A_FILE' | translate }}</label>
   <input type="file" id="image-file-input" (click)="this.imageInputRef.nativeElement.value = null"
          (change)="handleFile()"
-         class="image-uploader-file-input e2e-test-photo-upload-input"
+         class="image-receiver-file-input e2e-test-photo-upload-input"
          accept=".jpg,.jpeg,.png,.svg"
          [ngClass]="fileInputClassName" #imageInput>
-  <label *ngIf="isBlogPostThumbnailUploader"
-         class="blog-post-thumbnail-uploader d-flex flex-column flex-grow-1 text-center justify-content-center position-relative"
-         [ngClass]="{'blog-post-narrow-thumbnail-uploader': isWindowNarrow, 'image-uploader-is-narrow': blogDashboardPageService.imageUploaderIsNarrow, 'uploader-with-error-message': errorMessage}"
-         for="image-file-input">
-    <i class="fas fa-image image-uploader-no-thumbnail-icon"
-       [ngClass]="{'d-none': blogDashboardPageService.imageUploaderIsNarrow}">
-    </i>
-    <span *ngIf="!blogDashboardPageService.imageUploaderIsNarrow"
-          class="font-weight-light">
-      {{ 'I18N_BLOG_POST_THUMBNAIL_PICTURE_DRAG' | translate }}
-    </span>
-    <span *ngIf="blogDashboardPageService.imageUploaderIsNarrow"
-          class="font-weight-light">
-      {{ 'I18N_BLOG_POST_EDITOR_EDIT_THUMBNAIL_TEXT' | translate }}
```

Comment:
Oh, is this key needed, or can you remove it too?

Reply:
Yes its used in blog post editor. we can remove this once the whole issue is fixed and the blog post editor component is replaced via image-uploader.